raster-foundry/raster-foundry
# Remove blog/feed from api, frontend

## Overview
See title

### Checklist

- ~New tables and queries have appropriate indices added~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

### Notes
I didn't remove "FEED" from ActionType since I wasn't sure whether that's something we added for future support for Granary or for the blog feed. We never used it either way so... :man_shrugging: 

## Testing Instructions
- Verify that things compile
- Assemble the api server
- Make a request to /api/feed and verify that it 404's 
- Start and open the frontend, verify that the  feed is missing


Closes #5197
